ocs_ci/utility/tnf_hypervisor.py

```
@@ -0,0 +1,937 @@
+        try:
+            logger.info(f"Deleting auto-created security group: {sg_id}")
+            self.ec2_client.delete_security_group(GroupId=sg_id)
+            logger.info(f"Security group deleted: {sg_id}")
+        except Exception as e:
+            logger.warning(f"Failed to delete security group {sg_id}: {e}")
+
+    @staticmethod
+    def _shell_quote(s):
+        return "'" + s.replace("'", "'\"'\"'") + "'"
+
+    def _ssh_cmd(self, cmd, timeout=300, ignore_error=False, sudo=True):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_ssh_cmd`

(ANN202)